1/debian-10/rootfs/opt/bitnami/scripts/libmediawiki.sh

```diff
@@ -271,9 +272,15 @@ mediawiki_configure_smtp() {
 'host' => '${MEDIAWIKI_SMTP_HOST}',
 'IDHost' => '${MEDIAWIKI_SMTP_HOST_ID}',
 'port' => ${MEDIAWIKI_SMTP_PORT_NUMBER},
+EOF
+    if [ "$MEDIAWIKI_SMTP_AUTH" == "true" ]; then
+        cat >>"$MEDIAWIKI_CONF_FILE" <<EOF
 'username' => '${MEDIAWIKI_SMTP_USER}',
 'password' => '${MEDIAWIKI_SMTP_PASSWORD}',
-'auth' => true
+EOF
+    fi
+    cat >>"$MEDIAWIKI_CONF_FILE" <<EOF
+'auth' => ${MEDIAWIKI_SMTP_AUTH}
```

Comment:
In accordance with my above suggestion:

```suggestion
'auth' => $(php_convert_to_boolean "$MEDIAWIKI_ENABLE_SMTP_AUTH")
```